src/puppetlabs/dujour/version_check.clj

```diff
@@ -81,7 +81,8 @@
         version-data {:version current-version}
         request-body (-> request-values
                          (dissoc :product-name)
-                         (set/rename-keys {:agent-os :agent_os})
+                         (set/rename-keys {:agent-os :agent_os
+                                           :puppet-agent-versions :puppet_agent_versions})
```

Comment:
Aren't the arguments backwards there? The threading macro puts the `testmap` as the first argument.